### PR TITLE
add GHPR accessibility help

### LINKED ISSUE #5933: Add GHPR accessibility help dialog
cc @jooyoungseo, @alexr00 

Now that we have fixed some accessibility issues with the GHPR extension, JooYoung mentioned it would be good to give an overview of the GHPR view and helpful commands. 

Additionally, he suggested that we add some default keybindings so users don't have to configure them. @jooyoungseo, if you could share these, that would be great.

### PR DESCRIPTION
**Pull request by meganrogge**
fixes #5933

Once https://github.com/microsoft/vscode/pull/210116 is merged shortly, we can adopt this here 😄 . 

I am out tomorrow until Tuesday, so pls tweak and merge this.

I added most commands from the list JooYoung provided to the dialog. https://github.com/microsoft/vscode-pull-request-github/issues/5933#issuecomment-2064283137

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -662,13 +663,15 @@
 					"id": "pr:github",
 					"name": "%view.pr.github.name%",
 					"when": "ReposManagerStateContext != NeedsAuthentication && !github:resolvingConflicts",
-					"icon": "$(git-pull-request)"
+					"icon": "$(git-pull-request)",
+					"accessibilityHelpContent": "Helpful commands include: \n-Refresh PR List <keybinding:pr.refreshList>\n-GitHub Pull Requests: Focus on Issues View <keybinding:issue.refresh> \n-GitHub Pull Requests: Focus on Pull Requests View <keybinding:pr:github.focus>\n-GitHub Issues: Copy GitHub Permalink <keybinding:issue.copyGithubPermalink>\n-GitHub Issues: Create an Issue <keybinding:issue.createIssue>\n-GitHub Pull Requests: Create Pull Request <keybinding:command:pr.create>\n-GitHub Pull Requests: Start Review <keybinding:pr.startReview>\n-GitHub Pull Requests: Mark Pull Request Ready For Review <keybinding:pr.readyForReview>\n-GitHub Pull Requests: Approve <keybinding:review.approve>"
```
[the remark this answers is not in the document]
Comment (alexr00) [reply package.json:667]: Also, not all these commands support keybindings. The following should be removed:
- GitHub Pull Requests: Mark Pull Request Ready For Review
- GitHub Pull Requests: Start Review
- GitHub Pull Requests: Approve

[the remark this answers is not in the document]
Comment (alexr00) [reply package.json:667]: What is the syntax for these supposed to be? I see this, which isn't a valid markdown link:
```md
[Refresh issues](command:issue.refresh><keybinding:issue.refresh>
```

**Review by alexr00 — CHANGES_REQUESTED** [review]
Left some comments so you can see what I intend to change.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (alexr00) [reply package.json]: I'll remove the markdown and keybinding syntax for now.

**Review by alexr00 — APPROVED** [review]

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (meganrogge) [reply package.json]: that looks correct, there's an issue to improve upon this https://github.com/microsoft/vscode/issues/210665

[the remark this answers is not in the document]
Comment (alexr00) [reply package.json]: This looks unfinished to me. Why do we show raw markdown in an end-user features?

**Pull request merged**